config/yaml.go

```diff
@@ -85,6 +87,15 @@ func (c *Config) UnmarshalYAML(data []byte) error {
 		c.Push = v
 	}
 
+	if s.Locale != "" {
+		tag, err := language.Parse(s.Locale)
```

Comment:
Why `tag` ?

Reply:
The type was language.Tag.
I'm going to change that because it's confusing.

Reply:
Thank you!